configs/test.py

```diff
@@ -0,0 +1,28 @@
+from gem5.resources.resource import SuiteResource
```

Comment:
Needs license block

Reply:
This file will be removed, it was just for checking our initial implementation.